apps/api/src/app/auth/auth.controller.ts

```diff
@@ -167,6 +167,7 @@ export class AuthController {
     return await this.switchOrganizationUsecase.execute(command);
   }
 
+  // *DEPRECATED*
   @Post('/environments/:environmentId/switch')
```

Comment:
This endpoint is not necessary as switching is a frontend concern and no new auth_token is needed.